pkg/cmd/mirror/mirror.go

```diff
@@ -195,6 +411,19 @@ func dumpNewDepsBzl(listed map[string]listedModule, downloaded map[string]downlo
 	}
 	sort.Strings(sorted)
 
+	ctx := context.Background()
+	var client *storage.Client
+	if canMirror() {
+		var err error
+		client, err = storage.NewClient(ctx)
+		if err != nil {
+			return err
+		}
+	}
+
+	var waitingTasks int
+	uploadTasks := make(chan error)
```

Comment:
Should we limit the amount of parallel connections to the API, so we don't hit any rate limiting?

Reply:
I don't think there are any rate limits that apply here -- I mirrored every Go module all at once, and the program worked without any issues. In any case, from now on we won't have to mirror EVERY Go module, but only the new ones, which are much fewer.